packages/spanner/src/transaction.js

```diff
@@ -283,11 +314,28 @@ Transaction.prototype.commit = function(callback) {
  * @param {function} callback - The callback function.
  */
 Transaction.prototype.request = function(config, callback) {
+  var self = this;
+
   var reqOpts = extend({
     session: this.session.formattedName_
   }, config.reqOpts);
 
-  config.method(reqOpts, callback);
+  var gaxOptions = reqOpts.gaxOptions;
+  delete reqOpts.gaxOptions;
+
+  config.method(reqOpts, gaxOptions, function(err, resp) {
```

Comment:
@vkedia we need a little guidance here on if we should be discarding results from a stream that came in right before an aborted error.